.github/workflows/ci.yml

```diff
@@ -51,7 +50,6 @@ jobs:
         run: ./ci/script.sh
```

Comment:
Shouldn't this be split into build and lint steps, too?

Reply:
Didn't feel like it was necessary since the logs aren't a problem when linting.

Reply:
I'll fix it.